msal/application.py

```diff
@@ -700,6 +700,35 @@ def _validate_ssh_cert_input_data(self, data):
                     "you must include a string parameter named 'key_id' "
                     "which identifies the key in the 'req_cnf' argument.")
 
+    def acquire_token_by_refresh_token(self, refresh_token, scopes):
+        """Acquire token(s) based on a refresh token (RT) obtained from elsewhere.
+
+        In normal MSAL scenarios, you do NOT need to use this method.
```

Comment:
>In normal MSAL scenarios, you do NOT need to use this method. [](start = 8, length = 61)

[https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/blob/1cef43d185bb0a1dcc013d2ec232ae9d4c6c053a/src/client/Microsoft.Identity.Client/IByRefreshToken.cs](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/blob/1cef43d185bb0a1dcc013d2ec232ae9d4c6c053a/src/client/Microsoft.Identity.Client/IByRefreshToken.cs):
        /// Acquires an access token from an existing refresh token and stores it, and the refresh token, in
        /// the user token cache, where it will be available for further AcquireTokenSilent calls.
        /// This method can be used in migration to MSAL from ADAL v2, and in various integration
        /// scenarios where you have a RefreshToken available.
        /// See [https://aka.ms/msal-net-migration-adal2-msal2](https://aka.ms/msal-net-migration-adal2-msal2).

You can also get inspiration on the text from the Java API. Should be similar.

Please add aka.ms links to and create a wiki page allowing you to explain this in more details as well as updating content and linking to it easily when customers have questions.

Reply:
We probably do not need to create yet another wiki page. After this PR is released, we can modify the [existing doc](https://docs.microsoft.com/en-us/azure/active-directory/develop/migrate-python-adal-msal#migrate-existing-refresh-tokens-for-msal-python) accordingly.

Reply:
> After this PR is released, we can modify the [existing doc](https://docs.microsoft.com/en-us/azure/active-directory/develop/migrate-python-adal-msal#migrate-existing-refresh-tokens-for-msal-python) accordingly.

UPDATE:
A new [doc PR for the migration docs](https://github.com/MicrosoftDocs/azure-docs-pr/pull/116270) has been created just now.